docs/about.md

```diff
@@ -29,9 +27,9 @@ Scheduled backups are saved with the name `<SCHEDULE NAME>-<TIMESTAMP>`, where `
 
 ## Restores
 
-The **restore** operation allows you to restore all of the objects and persistent volumes from a previously created backup. Ark supports multiple namespace remapping--for example, in a single restore, objects in namespace "abc" can be recreated under namespace "def", and the objects in namespace "123" under "456".
+The **restore** operation allows you to restore all of the objects and persistent volumes from a previously created backup. You can also restore only a filtered subset of objects and persistent volumes. Ark supports multiple namespace remapping--for example, in a single restore, objects in namespace "abc" can be recreated under namespace "def", and the objects in namespace "123" under "456".
 
-Restored objects are labeled with the name `ark-restore=<BACKUP NAME>-<TIMESTAMP>`, where `<TIMESTAMP>` is formatted as *YYYYMMDDhhmmss*.
+Restored objects have the default name `ark-restore=<BACKUP NAME>-<TIMESTAMP>`, where `<TIMESTAMP>` is formatted as *YYYYMMDDhhmmss*. If you provided a custom name, it is used instead.
```

Comment:
This still isn't quite right. Each object that's restored has a Kubernetes label attached to it, whose key is `ark-restore`, and whose value is the name of the Ark restore (by default `BACKUP_NAME-TIMESTAMP`).  E.g. (see last line):

```
Name:           restic-test-workload-5b4c865bcb-ncvsh
Namespace:      default
Start Time:     Thu, 05 Jul 2018 09:10:08 -0700
Labels:         app=restic-test-workload
                ark-restore=a-20180705091011
```